src/components/saved-recipes/saved-recipes-list.js

```diff
@@ -12,6 +12,9 @@ const SavedRecipesList = () => {
                     className="form-control rounded-pill ps-5 text-dark"/>
             <i className="bi bi-search position-absolute wd-nudge-up text-dark"></i>
         </div>
+        <div className="ms-4 text-dark fw-bold">
+            <span>Total Saved Recipes: 3</span>
```

Comment:
Instead of hardcoding the recipe count, we can change the number 3 into {recipes.length} in line 16

Reply:
sounds good Ill change it!